### PR TITLE
Swiftlint 0.4.1 - Here we go again! 🤪🤪

### PR DESCRIPTION
**Pull request by kimdv**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/swiftlint/0.4.1/step.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ inputs:
     - junit
     - emoji
   reporter: xcode
-- opts:
+- opts: "no"
```
Comment (viktorbenei) [inline steps/swiftlint/0.4.1/step.yml]: you "fixed" this at the wrong place ;)

Comment (kimdv) [reply steps/swiftlint/0.4.1/step.yml]: Dooh... Think I'm going to bed soon... I'm an old man... 👴

Comment (viktorbenei) [reply steps/swiftlint/0.4.1/step.yml]: Don't worry, happens with the bests too ;)

**Comment by kimdv** [comment]
So a version `0.4.2` @viktorbenei ? 😄

**Comment by viktorbenei** [comment]
Yep, please 0.4.2 :)

**Pull request closed without merging**